docs/development-guide.md

```diff
@@ -73,6 +65,16 @@ The input variables, with their default values (some auto generated) are:
 - `docker_image_celeryworker`: Docker image for the celery worker. By default, based on your Docker image prefix.
 - `docker_image_frontend`: Docker image for the frontend. By default, based on your Docker image prefix.
 
+## Run Cookiecutter
+
+Go to the directory where you want to create your project and run:
+
+```bash
```

Comment:
This is running a bash script that installs + runs cookiecutter? Or two separate commands, one to install and one to run cookiecutter?

Reply:
One to install and one to run.